src/analysis/retail/evoker/augmentation/modules/Abilities.tsx

```diff
@@ -129,6 +130,7 @@ class Abilities extends CoreAbilities {
         spell: SPELLS.TREMBLING_EARTH_BUFF.id,
         category: SPELL_CATEGORY.OTHERS,
         gcd: null,
+        enabled: combatant.has4PieceByTier(TIERS.DF3) || combatant.has4PieceByTier(TIERS.DF4),
```

Comment:
these tier sets are disabled now, right? if so, this should be removed rather than gated by tier set.

Reply:
These tier sets are disabled, yes. I'll remove the old tier in a separate pull request.

There's currently an error which I couldn't work out exactly how to fix (still fairly new to this repo) where a few spells, including this one, show up in the cooldowns section when they clearly shouldn't. I've asked Voll about it, but in the meantime, requiring that tier set avoids the issue on this spell for anyone who doesn't have that 4pc equipped.